L/LLVM/common.jl

```diff
@@ -307,6 +307,11 @@ if [[ "${target}" == *apple* ]]; then
     fi
 fi
 
+if [[ "${target}" == aarch64-linux* ]]; then
+    CMAKE_C_FLAGS+=(-mno-outline-atomics)
+    CMAKE_CPP_FLAGS+=(-mno-outline-atomics)
+fi
```

Comment:
Should this be gated by version number again?

Reply:
This is gated on the GCC version used to build it

Reply:
But it isn't?

Reply:
Can we do that in the build_tarballs script?

Reply:
https://github.com/JuliaPackaging/Yggdrasil/blob/f59804dc452f9db4c1404c41562d2720e27d81ea/O/OpenBLAS/common.jl#L167